app/components/primer/alpha/segmented_control.html.erb

```diff
@@ -0,0 +1,9 @@
+<segmented-control>
+  <%= render Primer::BaseComponent.new(**@system_arguments) do %>
```

Comment:
Since the tag can't be customized, consider passing it directly instead of setting it in `system_arguments`.

```suggestion
  <%= render Primer::BaseComponent.new(tag: :ul, **@system_arguments) do %>
```